test/reliability.js

```diff
@@ -4,17 +4,22 @@ var path = require("path"),
     os = require("os"),
```

Comment:
The reason I changed this file was to compensate for the fact that the new tests I added increased the test suite's total run time